geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -2032,6 +2032,23 @@
    * Default: empty. All security components use basic (username/password) authentication
    */
   String SECURITY_AUTH_TOKEN_ENABLED_COMPONENTS = SECURITY_PREFIX + "auth-token-enabled-components";
+
+  /**
+   * The static String definition of the <i>"security-cn-auth-enabled"</i> property
+   * </p>
+   *
+   * <U>Description</U> This parameter only works if SSL with two-way handshake is enabled on
+   * all components and security manager is enabled. This property will determine if common name
+   * from client certificate will be used for authentication and authorization.
```

Comment:
Present tense would be clearer: 
This property allows the common name from the client certificate to be used for authentication and authorization.

Reply:
Thanks for the comments. I will rephrase it as you suggested.